### ISSUE TITLE
Android app expands map to external app with datetime of image instead of GPS coordinates 

### ISSUE THREAD

**Issue by samuel-tay** [immich-app/immich#3885]
### The bug

When trying to open a bigger map when viewing an image by clicking on the map icon, I can select an external map app, but the pre-filled location is the date and time of image instead of the GPS coordinates. 

### The OS that Immich Server is running on

Android 

### Version of Immich Server

v1.75.2

### Version of Immich Mobile App

v1.75.1 build 98

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
1. Open an image with GPS coordinates 
2. Expand image details by clicking the 'i' button at top right
3. Click the map button in the small map under location
4. Choose external mapping app
```

**Comment by shalong-tanwen** [comment]
I saw this while testing one of my recent changes as well, and since it was related to what I was working on, I fixed it in the same PR #3661

**Comment by shalong-tanwen** [comment]
This should be fixed in the latest version of the app
- #3911 

Feel free to re-open this if you still face the issue
